crates/noirc_frontend/src/lexer/lexer.rs

```diff
@@ -113,13 +130,13 @@ impl<'a> Lexer<'a> {
         single: Token,
         double: Token,
     ) -> SpannedTokenResult {
-        let start = self.position.mark();
+        let start = self.position;
 
         match self.peek_char_is(character) {
             false => Ok(single.into_single_span(start)),
             true => {
                 self.next_char();
-                Ok(double.into_span(start, start.forward()))
+                Ok(double.into_single_span(start))
```

Comment:
Does this record the span as 2 columns for double? so start = start and end = start +1

Reply:
It indeed does `(start, start + 1)` but this is wrong now since spans are no longer inclusive, good catch